### PR TITLE
Add must-gather logs to opendatahub

### PR DESCRIPTION
**Pull request by VaishnaviHire**
This is an initial script to get logs from pods running in the following namespaces:

- redhat-ods-operator
 - redhat-ods-applications
 - redhat-ods-monitoring

### CODE REVIEW

**Review by crobby — APPROVED** [review]
I tried this out and it seems to do what is expected.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM quay.io/openshift/origin-cli:latest
+
+# copy all collection scripts to /usr/bin
+COPY collection-scripts/* /usr/bin/
+
+ENTRYPOINT /usr/bin/gather
```
Comment (crobby) [inline Dockerfile:6]: Should have a newline here

File: README.md [hunk 2]
```diff
@@ -1 +1,36 @@
-# must-gather
\ No newline at end of file
+# must-gather for Open Data Hub
```
Comment (anishasthana) [inline README.md:1]: Should we change references of Open Data Hub to RHODS?

Comment (VaishnaviHire) [reply README.md:1]: I think yes.

**Comment by VaishnaviHire** [comment]
Merging this as all the comments are addressed

**Pull request merged**